commitglobal/votemonitor
# [83] Reusable datatable with example implementation

Closes #83 

### How was this tested
1. Run the C# api locally (web connects to `https://localhost:7123/api` by default)
2. Insert mock data in Postgres DB (a valid user, polling stations and observers can be found at `/src/Vote.Monitor.Domain/mock.sql`; mock auth expects `user`/`pass` to be valid credentials)
3. Run `pnpm install`
4. Run `pnpm run dev` and test inserted data
![paginated_table](https://github.com/commitglobal/votemonitor/assets/516929/97f203e9-0c77-4ab5-b3bc-0035f1786d93)

5. Run `pnpm run storybook` and check example `<DataTable />` component
![Storybook example](https://github.com/commitglobal/votemonitor/assets/516929/bb7bfa63-78b4-4142-ade9-0ce6241b83f9)
